openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -0,0 +1,91 @@
+from .constants import GROUP_TRAINING_MANAGERS, ADMIN, LEARNER, TRAINING_MANAGER
+
+from django.contrib.auth.models import User
+from django.db.models.query_utils import Q
+
+from .permissions import CanAccessPakXAdminPanel
+
+from rest_framework.response import Response
+from rest_framework import viewsets, status
+from rest_framework.decorators import permission_classes, authentication_classes
+from rest_framework.filters import OrderingFilter
+from rest_framework.authentication import SessionAuthentication, BasicAuthentication
+
+from student.models import UserAttribute
+from .serializers import UserSerializer
+
+
+class UserProfileViewSet(viewsets.ModelViewSet):
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
+    permission_classes = [CanAccessPakXAdminPanel]
+
+    queryset = None
+
+    serializer_class = UserSerializer
+
+    filter_backends = [OrderingFilter]
+
+    ordering = ['-id']
+
+    def create(self, request, *args, **kwargs):
+        if request.data['active_status'] is not None and request.data['active_status'] in [0, 1]:
+            return self.user_toggle_activation(bool(request.data["active_status"]), request.data["ids"])
+
+        return Response(status=status.HTTP_400_BAD_REQUEST)
+
+    def destroy(self, request, *args, **kwargs):
+
+        if User.objects.filter(id=kwargs['pk'], attributes__name="org", attributes__value=UserAttribute.get_user_attribute(self.request.user, "org")).update(is_active=False):
```

Comment:
Please keep the org checks in the base queryset variable and update it in respective functions accordingly.